processor/logdeduplicationprocessor/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+type: logdedup
+
+status:
+  class: processor
+  stability:
+    development: [logs]
+  distributions: []
+  warnings: []
+  codeowners:
+    active: [BinaryFissionGames, MikeGoldsmith]
```

Comment:
We'll want to add @djaglowski as a codeowner as well, since he's the sponsor for the component.